pkgs/development/libraries/qt-5/5.12/default.nix

```diff
@@ -8,7 +8,7 @@ top-level attribute to `top-level/all-packages.nix`.
 
 1. Update the URL in `pkgs/development/libraries/qt-5/$VERSION/fetch.sh`.
 2. From the top of the Nixpkgs tree, run
-   `./maintainers/scripts/fetch-kde-qt.sh > pkgs/development/libraries/qt-5/$VERSION/srcs.nix`.
+   `./maintainers/scripts/fetch-kde-qt.sh pkgs/development/libraries/qt-5/$VERSION/fetch.sh`.
```

Comment:
```suggestion
   `./maintainers/scripts/fetch-kde-qt.sh pkgs/development/libraries/qt-5/$VERSION`.
```

Reply:
Indeed there was inconsistency as for `srcs.nix` or `fetch.sh` but `fetch.sh` is needed due to:

https://github.com/doronbehar/nixpkgs/blob/106dc0f693d80dcf4ca68e49fb272f3cf5f83611/maintainers/scripts/fetch-kde-qt.sh#L7-L9

Reply:
Oh I misinterpreted the shell script. This works as well and looks better :).